### PR TITLE
add entry errors

### PR DESCRIPTION
**Pull request by Janet-Namutebi**
Hi @dennisja, When try to add an entry, I get these errors in intervals
```
Bad Request and the method not allowed errors
```
Right now it's the Bad request error:
```
HTTP/1.0 400 BAD REQUEST
Content-Type: text/html
Content-Length: 221
Server: Werkzeug/0.14.1 Python/3.5.2
Date: Mon, 24 Sep 2018 14:14:29 GMT

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>400 Bad Request</title>
<h1>Bad Request</h1>
<p>Failed to decode JSON object: Expecting property name enclosed in double quotes: line 1 column 2 (char 1)</p>
```

### CODE REVIEW

**Comment by dennisja** [comment]
Make sure you are sending it as json in post man

**Comment by dennisja** [comment]
Don't use `request.json`, it was deprecated use `request.get_json()`.

**Comment by dennisja** [comment]
screen shot 2018-09-24 at 17 39 56

**Comment by dennisja** [comment]
Does this fix your issue?

File: app.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+from flask import Flask, jsonify, request, make_response
+from datetime import datetime
+
+app = Flask(__name__)
+
+entries = [
```
Comment (dennisja) [inline app.py]: Would appreciate if you could use OOP
Look at the book example we did and think of how you can leverage OOP to solve this

File: app.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        'title':'bankaccount class',
+        'content':'class implementation corresponding to the provided tests',
+        'time':'2018-09-21 10:1:09.318212'
+    }
+        ]
+
+def get_timestamp():
+    return datetime.now().strftime(("%Y-%m-%d %H:%M:%S"))
+
+@app.route('/api/v1/entries', methods=['GET'])
```
Comment (dennisja) [inline app.py]: If you are using get only. You don't need the `methods=['GET']` drama

Comment (Janet-Namutebi) [reply app.py]: Alright, will change this

File: app.py (hunk 1 above)
Comment (Janet-Namutebi) [reply app.py]: I will look at it tomorrow

File: app.py [hunk 3]
```diff
@@ -37,17 +37,29 @@ def get_specific_entry(entryid):
...
 @app.route('/api/v1/entries', methods=['POST'])
 def add_entry():
-    entrycontent = {
-        'entryId':entries[-1]['entryId'] + 1,
-        'title': request.get_json['title'],
-        'content':request.get_json['content'],
-        'time':get_timestamp()
-        }
-    entries.append(entrycontent)
+    try:
```
Comment (dennisja) [inline app.py]: Oh No
There is a better way to do this

**Pull request merged**